python-backend/app/api/now_applications/models/activity_detail/etl_activity_detail.py

```diff
@@ -0,0 +1,19 @@
+import uuid, datetime
+
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.orm import validates
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.schema import FetchedValue
+from app.extensions import db
+
+from app.api.utils.models_mixins import AuditMixin, Base
+
+
+class ETLActivityDetail(Base):
```

Comment:
What does this table contain? Is this our way of associating existing Placers and Settling Ponds with new Activities? 

I don't think the name ETLActivityDetail is accurate in that case, since it would really have anything to do with the ETL process itself - except for the way its created.

Reply:
It contains a reference between our "detail" and the NROS/VFCBC placer/settling pond. Although it is not used by the SQL ETL scripts, is used by a type of "ETL" and more importantly has the same intent as the other ETL prefixed tables.